test/quantization/core/test_quantized_op.py

```diff
@@ -2880,6 +2880,18 @@ def test_qlinear_legacy(self, batch_size, input_channels, output_channels):
         self.assertEqual(Y_fp32, Y_fp32_ref,
                          msg="torch.ops.quantized.fbgemm_linear_dynamic results are off")
 
+    @given(
+        input_channels=st.integers(16, 32),
+        output_channels=st.integers(4, 8),
+        exponent=st.integers(0, 8))
+    def test_pack_unpack_fp16(self, input_channels, output_channels, exponent):
```

Comment:
actually maybe this could have a better name, like test_linear_prepack_fp16_numerics